base.pxt

```diff
@@ -647,27 +657,33 @@ window_area = 0.0
     rsVFPerTon = <%= fan_flow_per_cap %>
 
   <% if supply_duct_area > 0 %>
+    <% if duct_insulation_r_value_supply > 0 %>
```

Comment:
Handle the R=0 case in the function above (don't rely on parameter default for the air film since it shouldn't be an input parameter).

Reply:
Fixed.